extra/statistics-generator

```diff
@@ -131,6 +131,10 @@ p
 '
 }
 
+cleanup() {
+    sed  -e '/^Processed [0-9]\+ csets from [0-9]\+ developers$/d;/^[0-9]\+ employers found$/d'
```

Comment:
Just for future me, could you not do these as multiple sed expressions :pray:  :sweat_smile:

Reply:
I'll do it on two operations :ok_hand: